src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1496,7 +1496,7 @@ private BoundExpression BindNonMethod(SimpleNameSyntax node, Symbol symbol, Diag
 
                 case SymbolKind.Alias:
                     {
-                        var alias = symbol as AliasSymbol;
+                        var alias = (AliasSymbol)symbol;
                         symbol = alias.Target;
```

Comment:
Hmm.  Almost feel like we should have a code fixer for this type of pattern:

``` c# 
var x = expr as Type;
x.SomeMember
```

In the case `expr` isn't `Type` then this pattern will throw a `NullReferenceException`.  Much better to do the explicit cast and get the `InvalidCastException`.  Makes the problem much clearer. 

CC @kuhlenh